.editorconfig

```diff
@@ -33,7 +33,6 @@ dotnet_style_qualification_for_field = false:warning
 dotnet_style_qualification_for_method = false:warning
 dotnet_style_qualification_for_property = false:warning
 dotnet_style_require_accessibility_modifiers = for_non_interface_members:hint
-csharp_style_expression_bodied_methods = false:hint
```

Comment:
Because it disourages the use of expression bodied methods when really there should be no forcing either expression or statement style - if it can be an expression body no reason why it shouldn't be.

Reply:
true - was just wondering why here.

Reply:
Why where?

Reply:
why in this pr

Reply:
Because this is the first PR on 2.0 since the community contribution which changed the style - this must've slipped through the cracks in that PR.